pact_broker/config.ru

```diff
@@ -7,7 +7,14 @@ require_relative 'docker_configuration'
 require_relative 'pact_broker_resource_access_policy'
 
 dc = PactBroker::DockerConfiguration.new(ENV, PactBroker::Configuration.default_configuration)
-dc.pact_broker_environment_variables.each{ |key, value| $logger.info "#{key}=#{value}"}
+dc.pact_broker_environment_variables.each { |key, value|
```

Comment:
Good thinking. You'll find the code to do this is actually in `pact_broker/docker_configuration.rb`

```
def pact_broker_environment_variables
      pact_broker_environment_variable_names.sort.each_with_object({}) do | name, hash |
        hash[name] = name =~ /password/i ? "*****" : @env[name]
      end
    end
```

Would you mind updating that instead? Maybe just put it in a separate PR so I can merge the rest of it, which looks 👍🏽